rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init

```diff
@@ -174,6 +174,7 @@ BACKUP_SUFFIX="`date +%Y%m%d%H%M%S`.ldif"
 # Other
 TIMEOUT="60"    # Max time to stop process
 FD_LIMIT="2048" # Max file descriptor
+VM_LIMIT="108490720" # Max virtual memory
```

Comment:
Yes but at the beginning of the script the mdb size may not be computed already.

The simplest solution:

* `ulimit -v unlimited` at the beginning of the init script
* `ulimit -v unlimited` at the beginning of the migration script in postinstall

Virtual memory seems unlimited by default on modern OSes.